### ISSUE TITLE
New Ontology Request: Provisional Cell Ontology

### ISSUE THREAD

**Issue by shawntanzk** [OBOFoundry/OBOFoundry.github.io#1708]
Use this form to register a new ontology with the OBO Foundry. Please read the instructions provided here:
http://obofoundry.org/docs/NewOntologyRegistrationInstructions.html

## Ontology title
Provisional Cell Ontology

## Requested ID space
pcl

## Ontology location
https://github.com/obophenotype/provisional_cell_ontology

## Contact person
Name: David Osumi-Sutherland
Email address: davidos@ebi.ac.uk
GitHub username: dosumis

Name: Richard Scheuermann
Email address: rscheuermann@jcvi.org
GitHub username: scheuerm

## Issue tracker
https://github.com/obophenotype/provisional_cell_ontology/issues

## Ontology license

- [ ] CC0 (public domain)
- [x] CC-BY (version 3 or later)
- [ ] Other: *please specify*

## Available ontology formats
owl
obo
json

## What domain is the ontology intended to cover?
Cell types that are provisionally defined by experimental techniques such as single cell or single nucleus transcriptomics rather than a straightforward & coherent set of properties.  All terms subclass to conventionally defined terms in CL. If more data emerges that allows more conventional definitions, terms may migrate to CL.

## Related OBO Foundry ontologies
Cell Ontology

## Intended use/related projects
Annotations for single cell transcriptomics projects. 
Search and navigation of single cell transcriptomics data explorer (see DOI:10.1101/2021.10.10.463703)

## Data source
Single Cell/Nucleus Transcriptomics experiments.

## OBO Foundry pre-registration checklist

To be considered for inclusion in the OBO Foundry, an ontology must meet certain requirements, as described in in [the registration process instructions](http://obofoundry.org/docs/Policy_for_OBO_namespace_and_associated_PURL_requests.html) and the [registration review checklist](https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/docs/RegistrationChecklist.md). To ensure you are aware of some of its key points, please review the checklist below.

You can either check a box by submitted the request first and then using the GitHub interface, or replacing the `- [ ]` by `- [X]` in the following.

- [X] I have read an understood [the registration process instructions](http://obofoundry.org/docs/Policy_for_OBO_namespace_and_associated_PURL_requests.html) and the [registration checklist](https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/docs/RegistrationChecklist.md) 
- [X] There is no other ontology in the OBO Foundry which would be an appropriate place for my terms. If there was, I have contacted the editors, and we decided in mutual agreement that a separate ontology is more appropriate.
- [X] My ontology has a specific release file with a version IRI and a dc:license annotation, serialised in RDF/XML.
- [X] I understand that term definitions, while not mandatory, are key to understanding the intentions of a term especially when the ontology is used in curation. I made sure that a reasonable majority of terms in my ontology have definitions, using the [IAO:0000115](http://www.ontobee.org/ontology/IAO?iri=http://purl.obolibrary.org/obo/IAO_0000115) property.
- [X] For every term in my ontology, I checked whether another OBO Foundry ontology has one with the same meaning. If so, I re-used that term directly (not by cross-reference, by directly using the IRI).
- [X] For all relationship properties (Object and Data Property) I checked whether [RO](http://www.ontobee.org/ontology/catalog/RO?iri=http://www.w3.org/2002/07/owl%23ObjectProperty) includes an appropriate one. I understand that aligning with RO is an essential part of the overall alignment between OBO ontologies!
- [X] For the selection of appropriate annotation properties, I looked at [OMO](http://www.ontobee.org/ontology/catalog/OMO?iri=http://www.w3.org/2002/07/owl%23AnnotationProperty) first. I understand that aligning ontology metadata and term-level metadata is essential for cross-integration of OBO ontologies. 
- [X] If I was not sure about the meaning of any of the checkboxes above, I have consulted with a member of the OBO Foundry for advice.

## Metadata

Please fill in the following metadata record which will be used by the OBO Foundry website. Note that the values shown are just examples, for example `yourfourletterid` could be something like `aism`, `cohm`, `mondo` (it does not have to be four letters). `your_domain_like_for_example_anatomy` could be simply `anatomy`, and the license should be whatever your actual license is. An example can be found [here](https://github.com/OBOFoundry/OBOFoundry.github.io/edit/master/ontology/amphx.md), but you really only need to fill in the metadata mentioned here.

```
id: pcl
title: "Provisional Cell Ontology"
contact:
  email: davidos@ebi.ac.uk
  label: David Osumi-Sutherland
  email: rscheuermann@jcvi.org
  label: Richard Scheuermann
description: Cell types that are provisionally defined by experimental techniques such as single cell or single nucleus transcriptomics rather than a straightforward & coherent set of properties. 
domain: obophenotype
homepage: https://github.com/obophenotype/provisional_cell_ontology
products:
  - id: pcl.owl
  - id: pcl.obo
  - id: pcl.json
  - id: pcl-base.owl
  - id: pcl-base.obo
  - id: pcl-base.json
  - id: pcl-full.owl
  - id: pcl-full.obo
  - id: pcl-full.json
  - id: pcl-simple.owl
  - id: pcl-simple.obo
  - id: pcl-simple.json
dependencies:
  - id: pr
  - id: go
  - id: uberon 
  - id: ro
  - id: pato
  - id: ncbitaxon
  - id: bfo
  - id: cl
  - id: omo
  - id: nbo
  - id: chebi
  - id: so
tracker: https://github.com/obophenotype/provisional_cell_ontology/issues
license:
  url: http://creativecommons.org/licenses/by/4.0/
  label: CC-BY 4.0
usages:
  - user: https://biccn.org/
    description: This ontology will be used to annotate cell types in single cell transcriptomics data, with an initial focus on the brain. It is also used to drive search and navigation in a cell type data explorer web application. 
```

**Comment by shawntanzk** [comment]
@dosumis tagging to ping you

**Comment by lubianat** [comment]
@shawntanzk interesting proposal. 
Any comments on how definitions will be set? 
I imagine there won't be complete definitions in a rigorous sense (i.e. necessary and sufficient). 
How do you plan on handling that? I see: 
- https://github.com/obophenotype/provisional_cell_ontology/issues/11

But I am under the impression that issue is more about _scope_ than definitions per se.

**Comment by matentzn** [comment]
@shawntanzk thank you for you submission. I have updated your metadata according to our standards, please use that moving forward:

```
id: pcl
      mirror_from: https://raw.githubusercontent.com/obophenotype/provisional_cell_ontology/master/pcl-base.owl
      title: "Provisional Cell Ontology"
      contact:
        email: davidos@ebi.ac.uk
        label: David Osumi-Sutherland
        github: dosumis
      description: Cell types that are provisionally defined by experimental techniques such as single cell or single nucleus transcriptomics rather than a straightforward & coherent set of properties. 
      domain: phenotype
      homepage: https://github.com/obophenotype/provisional_cell_ontology
      products:
        - id: pcl.owl
        - id: pcl.obo
        - id: pcl.json
        - id: pcl-base.owl
        - id: pcl-base.obo
        - id: pcl-base.json
        - id: pcl-full.owl
        - id: pcl-full.obo
        - id: pcl-full.json
        - id: pcl-simple.owl
        - id: pcl-simple.obo
        - id: pcl-simple.json
      dependencies:
        - id: pr
        - id: go
        - id: uberon 
        - id: ro
        - id: pato
        - id: ncbitaxon
        - id: bfo
        - id: cl
        - id: omo
        - id: nbo
        - id: chebi
        - id: so
      tracker: https://github.com/obophenotype/provisional_cell_ontology/issues
      license:
        url: http://creativecommons.org/licenses/by/4.0/
        label: CC BY 4.0
      usages:
        - user: https://biccn.org/
          description: This ontology will be used to annotate cell types in single cell transcriptomics data, with an initial focus on the brain. It is also used to drive search and navigation in a cell type data explorer web application. 
      repository: https://github.com/obophenotype/provisional_cell_ontology
      preferredPrefix: PCL
```

You can see your dashboard checks here:

https://obofoundry.org/obo-nor.github.io/dashboard/index.html

As per OBO guidelines no check may fail apart from usages. You don't fail any checks, so you are good to go! Impressive! (The warning on the license is a minor issue I have fixed for you in the above metadata, so no need to act on it).

Please, as per new guidelines:
- [ ] Confirm you have no namespace clash with [BioPortal](http://bioportal.bioontology.org/ontologies)
- [ ] Confirm you have no namespace clash with [bioregistry](https://bioregistry.io/)

We will assign a reviewer at our next operations call, and will come back with a decision 14 days after that. Rough timeline: end of January, hope that is ok.

**Comment by shawntanzk** [comment]
@lubianat 
Definitions are something we struggled with - I think there needs to be a wider conversation about duplicate definitions as with ssRNAseq cell typing - like you mentioned, necessary and sufficient definitions are difficult. Currently only the Brain Data Standards project is in PCL and we solved this by adding in the definition the ID of the cell type in the data. We are also fortunate to have a set of gene markers that that is sufficient to differentiate cell types from each other. However, we do not expect that this would the case for every set of data in the future. 

As for scope, I think currently we are using what we wrote in the description: "Cell types that are provisionally defined by experimental techniques such as single cell or single nucleus transcriptomics rather than a straightforward & coherent set of properties." I do think it is worth a discussion (maybe among the cell ontology community) on what we consider a coherent set of properties 

tl;dr version: 
1) need to discuss with wider community on the need for unique defs
2) scope for pcl is cells that are defined by experimental techniques rather than set of properties

**Comment by shawntanzk** [comment]
@matentzn 
Thanks so much! 
Bioportal has PCL in it already, but we plan to replace that with our version - the people who created that are part of our project - do we need to replace that with ours first before I check that off?

**Comment by matentzn** [comment]
If it is conceptually the same ontology, then no, you don't have to. Its fine! Is it conceptually the same ontology?

**Comment by shawntanzk** [comment]
@matentzn it's the exact same ontology, we have just upgraded and updated it 
in that case, we are good to go for both :) Thanks!

**Comment by matentzn** [comment]
@ OFOC, this is now ready for review - please propose reviewers.

**Comment by scheuerm** [comment]
@lubianat and @shawntanzk I'm not sure I agree that the question about definitions is something we are struggling with.  In my view, we have a very clear approach for describing the necessary and sufficient conditions within the cell type definitions.  To me, defining cell types based on experimental results is much more rigorous than defining them by "a straightforward & coherent set of properties."  To be honest, I don't even know what that means.

**Comment by shawntanzk** [comment]
@scheuerm I agree that we have a pretty good and clear approach with the BDS - I guess I'm thinking in terms of PCL in general, where future transcriptomic cell types might be included in. These might not come with as clear definitions - I think @dosumis has examples from the fly world where it is just clustered by algorithms which makes having unique definitions a bit more tricky (but I might be wrong). Even in BDS, for cross-species terms for example (eg Lamp5 C_1 vs Lamp5 C_2) - to create a definition that differentiates those nodes was tricky and we solved it by saying that they represent cluster XXXX in dataset. NS forest markers does make it a bit easier on our part, but I guess we cannot expect all PCL terms in the future to have those. 

> To me, defining cell types based on experimental results is much more rigorous than defining them by "a straightforward & coherent set of properties." To be honest, I don't even know what that means.

I do agree with this in general, but I think practically, it still comes with some difficulties, especially in integrating with "traditionally" defined cell types - I guess the way we have done it is an example of how to approach it - would be interesting to see what the wider community thinks of our approach. But yes, I too feel like "a straightforward & coherent set of properties." is vague and I think something we should probably discuss as a community is what should go into CL and what should go into PCL - it is something I personally am struggling with. 

Hope my thoughts make sense - am a bit wonky from COVID booster heh

**Comment by nlharris** [comment]
Are there still things that @shawntanzk needs to do for this ontology to be eligible for OBO Foundry?

**Comment by shawntanzk** [comment]
> Are there still things that @shawntanzk needs to do for this ontology to be eligible for OBO Foundry?

From my understanding of the procedure, I think the next step for this is to be reviewed by the foundry and discussed in their next meeting

**Comment by nlharris** [comment]
On the agenda for tomorrow's meeting.

**Comment by shawntanzk** [comment]
Thanks @nlharris :)

**Comment by nlharris** [comment]
Approved in OBO Operations meeting, 25-Jan-2022!

**Comment by shawntanzk** [comment]
WOHOO! thanks so much @nlharris :)

**Comment by matentzn** [comment]
@shawntanzk 

Next steps:
1. Make a PR analogous to this one: https://github.com/OBOFoundry/OBOFoundry.github.io/pull/1673/files
2. Create a file like https://github.com/OBOFoundry/purl.obolibrary.org/blob/master/config/aism.yml (in PR)

 Ping me on slack if you need help.

**Comment by scheuerm** [comment]
Excellent!

--------------------------------------------
Richard H. Scheuermann, Ph.D.
Director, La Jolla Campus
J. Craig Venter Institute
4120 Capricorn Ln.
La Jolla, CA 92037

***@***.******@***.***>
858-200-1876

www.jcvi.org<http://www.jcvi.org>
https://www.jcvi.org/about/richard-scheuermann


From: Nomi Harris ***@***.***>
Reply-To: "OBOFoundry/OBOFoundry.github.io" ***@***.***>
Date: Tuesday, January 25, 2022 at 9:05 AM
To: "OBOFoundry/OBOFoundry.github.io" ***@***.***>
Cc: "Richard H. Scheuermann" ***@***.***>, Mention ***@***.***>
Subject: Re: [OBOFoundry/OBOFoundry.github.io] New Ontology Request: Provisional Cell Ontology (Issue #1708)


Approved in OBO Operations meeting, 25-Jan-2022!

—
Reply to this email directly, view it on GitHub<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FOBOFoundry%2FOBOFoundry.github.io%2Fissues%2F1708%23issuecomment-1021411739&data=04%7C01%7Crscheuermann%40jcvi.org%7Cce155465081a4d4484a508d9e024ec29%7C24d967f13ed84448baa6560ec572acb3%7C0%7C0%7C637787271475705946%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=bMbjv0EZYSoNGI3LUW93Um9TlRbGehFnuMmW%2FRfg9HA%3D&reserved=0>, or unsubscribe<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FABPP4NEIXP57ERRD3P6BJ6TUX3J6NANCNFSM5KI2464Q&data=04%7C01%7Crscheuermann%40jcvi.org%7Cce155465081a4d4484a508d9e024ec29%7C24d967f13ed84448baa6560ec572acb3%7C0%7C0%7C637787271475705946%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=nqRflr3dOoPm307C0%2FphEB0pn0DCGChJ3VPJriMpE10%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Crscheuermann%40jcvi.org%7Cce155465081a4d4484a508d9e024ec29%7C24d967f13ed84448baa6560ec572acb3%7C0%7C0%7C637787271475705946%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=Oc6z3lyrK%2FnjzHRX5rMLzNVqM0NatiG5BnotRysQIeg%3D&reserved=0> or Android<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Crscheuermann%40jcvi.org%7Cce155465081a4d4484a508d9e024ec29%7C24d967f13ed84448baa6560ec572acb3%7C0%7C0%7C637787271475705946%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=xkHDla8S4VDV0MJboWAACnSh8Jix7GN%2Ft54rLmluNwE%3D&reserved=0>.

**Comment by patrick-lloyd-ray** [comment]
Fantastic!

**Comment by matentzn** [comment]
Thank you @shawntanzk, next the PR on the PURL registry!

**Comment by nlharris** [comment]
Are we waiting for @shawntanzk to do that PR or is that something OBO Foundry people need to do?

**Comment by shawntanzk** [comment]
whoops, sorry, should close this, thanks @nlharris for keeping track, and thanks everyone for the help with getting PCL through to the foundry!
